### ISSUE TITLE
“Uncommented issue” is not always a problem

### ISSUE THREAD

**Issue by Kwpolska** [basicallydan/forkability#57]
The maintainers might create issues to track their task, as a to-do list.

**Comment by basicallydan** [comment]
Agreed! In fact, with this in mind I originally set up the code so that if the user who opened the issue is the same as the user who owns the repo, it won't count as a problem. I think in 90% of cases, the to-do list use case will be fine. For instance, [this uncommented issue](https://github.com/basicallydan/forkability/issues/10) does not show up as a problem with Forkability because I created it.

However, I did not account for the possibility that multiple people may be maintainers, or that the repo may be owned by an organisation. Do you mind if I edit your issue slightly to reflect these things?

**Comment by Kwpolska** [comment]
Do it.

-- 
Chris Warrick <https://chriswarrick.com/>
